### PR TITLE
Translate metainfo

### PR DESCRIPTION
**Pull request by cewbdex**
I made this separate, because the editor @Alexander-Wilms suggested reordered the file. Open to any suggestions.

### CODE REVIEW

**Comment by Alexander-Wilms** [comment]
Can you check the option "Use tabs instead of spaces" in the jdAppStreamEdit options to reduce whitespace changes?

**Comment by cewbdex** [comment]
Done, looks better.

File: launcher/eu.vcmi.VCMI.metainfo.xml [hunk 1]
```diff
@@ -1,90 +1,97 @@
...
-		<control>keyboard</control>
-		<control>pointing</control>
-		<control>touch</control>
-	</recommends>
 	<categories>
 		<category>Game</category>
 		<category>StrategyGame</category>
 	</categories>
-	<releases>
-		<release version="1.4.1" date="2023-12-12" />
```
Comment (Alexander-Wilms) [inline launcher/eu.vcmi.VCMI.metainfo.xml:71]: This release disappeared

Comment (cewbdex) [reply launcher/eu.vcmi.VCMI.metainfo.xml:71]: Sorry I missed that, rebased the file from another branch and this disappeared.

**Comment by cewbdex** [comment]
I have changed to merge into beta, so it an be in 1.4.2.

**Comment by Laserlicht** [comment]
can you also include https://github.com/cewbdex/vcmi/pull/1 :)

File: launcher/eu.vcmi.VCMI.metainfo.xml [hunk 2]
```diff
@@ -1,90 +1,109 @@
...
+			<li xml:lang="de">Vollständige Spielmechanik</li>
 			<li>Almost all objects, abilities, spells and other content</li>
+			<li xml:lang="cs">Skoro všechny předměty, schopnosti, kouzla a ostatní obsah</li>
+			<li xml:lang="de">Fast alle Objekte, Fähigkeiten, Zaubersprüche und andere Inhalte</li>
 			<li>Basic battle AI and adventure AI</li>
+			<li xml:lang="cs">Základní AI boje a mapy světa</li>
+			<li xml:lang="de">Grundlegende Kampf- und Abenteuer-KI</li>
 			<li>Many GUI improvements: high resolutions, stack queue, creature window</li>
+			<li xml:lang="cs">Mnoho vylepšení rozhraní: vyšší rozlišení, fronta oddílů a okno bojovníků</li>
+			<li xml:lang="de">Viele GUI-Verbesserungen: hohe Auflösungen, Truppenwarteschlange, Kreaturenfenster</li>
```
Comment (Alexander-Wilms) [inline launcher/eu.vcmi.VCMI.metainfo.xml:28]:
```suggestion
			<li xml:lang="de">Viele GUI-Verbesserungen: Hohe Auflösungen, Truppenwarteschlange, Kreaturenfenster</li>
```

File: launcher/eu.vcmi.VCMI.metainfo.xml [hunk 3]
```diff
@@ -1,90 +1,109 @@
...
 			<li>Advanced and easy mod support - add new towns, creatures, heroes, artifacts and spells without limits or conflicts</li>
+			<li xml:lang="cs">Pokročilá a jednoduchá podpora modifikací - přidání nových měst, bojovníků, hrdinů, artefaktů a kouzel bez limitů a konfliktů</li>
+			<li xml:lang="de">Erweiterte und einfache Mod-Unterstützung - fügt neue Städte, Kreaturen, Helden, Artefakte und Zaubersprüche ohne Einschränkungen oder Konflikte hinzu</li>
```
Comment (Alexander-Wilms) [inline launcher/eu.vcmi.VCMI.metainfo.xml:31]:
```suggestion
			<li xml:lang="de">Erweiterte und einfache Mod-Unterstützung - füge neue Städte, Kreaturen, Helden, Artefakte und Zaubersprüche ohne Einschränkungen oder Konflikte hinzu</li>
```

**Pull request merged**